site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -13,7 +13,7 @@
         <div onclick='updateDropdownStates($(this).find(".dropdown-arrow").hasClass("down"), "today_polls_dropdown", "{{ base_url }}")' class="dropdown-bar">
```

Comment:
Should this line be updated, now that `dropdown-arrow` has been removed? Same comment for line 76 and 141 in this file.

Reply:
I believe that whole onclick event was unnecessary so I have updated my PR.  I don't believe that it was doing anything to begin with because it had single quotes around it which messes up the parsing on some browsers.